### PR TITLE
Create CasualUC1.md

### LINKED ISSUE #34: Create casual UC1

### PR DESCRIPTION
**Pull request by Hjordrup**
Co-authored-by: MBRZealand <70659124+MBRzealand@users.noreply.github.com>

Closes #34

### CODE REVIEW

File: 02 Requiements/UseCases/UseCase1/CasualUC1.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Use case 1: Create Order 
+
+**Actors:**  
+Consultant. 
+
+**Main Succeses Scenario:**  
+
+The application prompts the user to specify the dimensions for the New Yorker wall.
+A Consultant specifies dimensions for New Yorker wall.
+A Consultant confirms dimensions for New Yorker wall.
```
Comment (Bombohog) [inline 02 Requiements/UseCases/UseCase1/CasualUC1.md:10]: Do the user confirm?

File: 02 Requiements/UseCases/UseCase1/CasualUC1.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+The application displays a slider for number of glass panels allowed vertically.
+The application displays a slider for number of glass panels allowed horizontally.
+A Consultant specifies the number of glass panels vertically.
+A Consultant specifies the number of glass panels Horizontally.
+The application displays the dimensions of the glass panels. The application redirects the Consultant to Customize Order.
```
Comment (Bombohog) [inline 02 Requiements/UseCases/UseCase1/CasualUC1.md:15]: The application redirects? and shouldnt it be on another line.

File: 02 Requiements/UseCases/UseCase1/CasualUC1.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+The application displays a error message, prompting the consultant to use a glass panel amount above the minimum value. 
+
+The Consultant chooses a number of glass panels vertically, which results in a glass panel height above maximum value.  
+The application displays a error message, prompting the consultant to use a glass panel amount below the maximum value. 
+
+The Consultant chooses a number of glass panels horizontally, which results in a glass panel width below minimum value.  
+The application displays a error message, prompting the consultant to use a glass panel amount above the minimum value.  
+
+The Consultant chooses a number of glass panels horizontally, which results in a glass panel width above maximum value.  
+The application displays a error message, prompting the consultant to use a glass panel amount below the maximum value.
```
Comment (Bombohog) [inline 02 Requiements/UseCases/UseCase1/CasualUC1.md:35]: Is this possible if we plan on making the algorithm only let the slider select usable number of panels.

**Review by Bombohog — CHANGES_REQUESTED** [review]
Look at these

File: 02 Requiements/UseCases/UseCase1/CasualUC1.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# Use case 1: Create Order 
+
+**Actors:**  
+Consultant.
```
Comment (Madictian) [inline 02 Requiements/UseCases/UseCase1/CasualUC1.md:4]: Remove "." in front of consultant

File: 02 Requiements/UseCases/UseCase1/CasualUC1.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
+# Use case 1: Create Order 
+
+**Actors:**  
+Consultant. 
+
+**Main Succeses Scenario:**  
+
+The application prompts the user to specify the dimensions for the New Yorker wall.
```
Comment (Madictian) [inline 02 Requiements/UseCases/UseCase1/CasualUC1.md:8]: The user isn't prompted?

**Review by Madictian — CHANGES_REQUESTED** [review]

File: 02 Requiements/UseCases/UseCase1/CasualUC1.md (hunk 1 above)
Comment (Hjordrup) [reply 02 Requiements/UseCases/UseCase1/CasualUC1.md:10]: fixed

**Review by Bombohog — APPROVED** [review]
I like it

**Pull request merged**